test/multiverse/lib/multiverse/suite.rb

```diff
@@ -287,9 +287,9 @@ def generate_gemfile(gemfile_text, env_index, local = true)
         f.puts "gem 'warning'"
 
         if debug
-          f.puts "gem 'pry', '~> 0.14'"
-          f.puts "gem 'pry-nav'"
-          f.puts "gem 'pry-stack_explorer', platforms: :mri"
+          f.puts "gem 'pry', '~> 0.14'" if ENV['ENABLE_PRY']
+          f.puts "gem 'pry-nav' if ENV['ENABLE_PRY']"
+          f.puts "gem 'pry-stack_explorer', platforms: :mri' if ENV['ENABLE_PRY']"
```

Comment:
I think the quotes on these are slightly off what we're looking for here:
```suggestion
          f.puts "gem 'pry-nav'" if ENV['ENABLE_PRY']
          f.puts "gem 'pry-stack_explorer', platforms: :mri" if ENV['ENABLE_PRY']
```

If the double quotes come after `ENV['ENABLE_PRY']` that part will be evaluated as a string instead of as an `if` statement.